gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/internal/ComposeProjectProperties.kt

```diff
@@ -22,7 +22,7 @@ internal object ComposeProperties {
     internal const val MAC_NOTARIZATION_PASSWORD = "compose.desktop.mac.notarization.password"
     internal const val MAC_NOTARIZATION_TEAM_ID_PROVIDER = "compose.desktop.mac.notarization.teamID"
     internal const val CHECK_JDK_VENDOR = "compose.desktop.packaging.checkJdkVendor"
-    internal const val ALWAYS_GENERATE_RESOURCE_ACCESSORS = "compose.resources.always.generate.accessors"
+    internal const val ENABLE_MULTIMODULE_RESOURCES = "compose.resources.multimodule.enable"
```

Comment:
Is it about publication and depending on resource library, or only about publication?

Reply:
It is about a publication and a multimoduleness (very close things in the gradle)

Reply:
What happen if it is `false`, and users add a dependency on a library with resources?

Reply:
oh, you meant a library with resources (not the resource's library!). No, it requires the flag